src/templates/apMesa.tpl.html

```diff
@@ -21,7 +21,7 @@
               ng-if="column.sort"
               title="This column is sortable. Click to toggle sort order. Hold shift while clicking multiple columns to stack sorting."
               class="sorting-icon {{ getSortClass( sortDirection[column.id] ) }}"
-            ></span>
+            ><span ng-bind="sortPriority[column.id]" class="sort-priority" ng-if="sortPriorityShow && sortPriority[column.id]"></span></span>
```

Comment:
Let's split out the `class="sorting-icon {{ getSortClass( sortDirection[column.id] ) }}"` into its own span and make it inside the pre-existing span (starting on line 20). Then make your new span a sibling of it, e.g.:

```html
<span ng-if="column.sort" title="...">
  <span class="sorting-icon {{ getSortClass( sortDirection[column.id] ) }}"></span>
  <span ng-bind="sortPriority[column.id]" class="sort-priority" ng-if="sortPriorityShow && sortPriority[column.id]"></span>
</span>
```